engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -95,11 +95,13 @@ <h4 translate>Sync Settings</h4>
           <input id="upstream_username"
                  name="upstream_username"
                  type="name"
+                 autocomplete="off"
```

Comment:
Was this changed as a potential safeguard around caching angular could be doing?

Reply:
This is 'the' way of disabling auto complete on a form input, however chrome no longer seems to support this.  My thinking is that we should prefer to have it autocomplete disabled for this field and do our best to do that where we can.   Although maybe we should allow it?

Reply:
To clarify, this problem is solved even without this line, but it seemed like a good idea to me

Reply:
gotcha. makes sense to not autocomplete since the values are getting saved